docs/addons/aws-for-fluent-bit.md

```diff
@@ -0,0 +1,22 @@
+# AWS for Fluent Bit
+
+Fluent Bit is an open source log processor and forwarder which allows you to collect data like metrics and logs from different sources, enrich them with filters and send them to multiple destinations.
+
+AWS provides a Fluent Bit image with plugins for both CloudWatch Logs and Kinesis Data Firehose. The [AWS for Fluent Bit image](https://gallery.ecr.aws/aws-observability/aws-for-fluent-bit) is available on the Amazon ECR Public Gallery. For more details, see [AWS for Fluent Bit GitHub repository](https://github.com/aws/aws-for-fluent-bit).
+
+## Usage
+
```

Comment:
Can we add a section on Configuration and reference the values from the charts that you referenced in the docs (https://github.com/aws/eks-charts/tree/master/stable/aws-for-fluent-bit) in the doc as well, similar to what we did for e.g. Nginx[here](https://github.com/aws-quickstart/ssp-amazon-eks/blob/main/docs/addons/nginx.md#configuration)?